safe/definitions/exposure.py

```diff
@@ -229,7 +230,7 @@
         'line'
     ],
     'size_unit': unit_metres,
-    'units': [unit_metres],
+    'units': [unit_metres, unit_kilometres],
```

Comment:
@lucernae Yes i can undertsand. I was doing by increasing order, it's also natural for me.
But currently, I have a problem if we have more than 2 units because I my code is doing only one conversion. It's low priority to have more than 2 units, I have some other blockers for now. Is-it ok like this ?